src/anyio/abc/_eventloop.py

```diff
@@ -343,6 +343,11 @@ async def wait_readable(cls, obj: HasFileno | int) -> None:
     async def wait_writable(cls, obj: HasFileno | int) -> None:
         pass
 
+    @classmethod
+    @abstractmethod
+    def notify_closing(cls, obj: HasFileno | int) -> None:
```

Comment:
Should we switch these to `FileDescriptorLike` too?

Reply:
Yep